bin/install-theos

```diff
@@ -61,6 +61,34 @@ fi
 
 
 # The work
+ver_compare() {
```

Comment:
Can we assume that dpkg is installed on host systems? If so, we can use `dpkg --compare-versions` instead of hand-rolling our own.

Reply:
Seems --compare-versions was added in dpkg [v1.2.1](https://github.com/oerdnj/dpkg/blob/347f239062f867b6112178338f983803906f5099/debian/changelog#L11104) and  Telesphoreo started at [v1.13.x](http://apt-origin.saurik.com/debs/), so it should be fine?